### PR TITLE
Add package installation instructions to the docs and README

### PR DESCRIPTION
**Pull request by adamperkowski**
## Type of Change
- [x] Documentation update

## Markdown preview

---

## ⬇️ Installation

Linutil is also available as a package in various repositories:

[Packaging status](https://repology.org/project/linutil/versions)

<details>
  <summary>Arch Linux</summary>

Linutil can be installed on [Arch Linux](https://archlinux.org) with three different [AUR](https://aur.archlinux.org) packages:

- `linutil` - Stable release compiled from source
- `linutil-bin` - Stable release pre-compiled
- `linutil-git` - Compiled from the last commit (not recommended)

by running:

```bash
git clone https://aur.archlinux.org/<package>.git
cd linutil
makepkg -si
```

Replace `<package>` with your preferred package.

If you use [yay](https://github.com/Jguer/yay), [paru](https://github.com/Morganamilo/paru) or any other [AUR Helper](https://wiki.archlinux.org/title/AUR_helpers), it's even simplier:

```bash
paru -S linutil
	
adamperkowski marked this conversation as resolved.
```

Replace `paru` with your preferred helper and `linutil` with your preferred package.
	
adamperkowski marked this conversation as resolved.

</details>

<details>
  <summary>Cargo</summary>

Linutil can be installed via [Cargo](https://doc.rust-lang.org/cargo) with:

```bash
cargo install linutil
```

Note that crates installed using `cargo install` require manual updating with `cargo install --force` (update functionality is [included in LinUtil](https://christitustech.github.io/linutil/userguide/#applications-setup))

</details>

---

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -13,6 +15,55 @@ To get started, run the following command in your terminal:
...
+
+```bash
+git clone https://aur.archlinux.org/<package>.git
+cd linutil
+makepkg -si
+```
+
+Replace `<package>` with your preferred package.
+
+If you use [yay](https://github.com/Jguer/yay), [paru](https://github.com/Morganamilo/paru) or any other [AUR Helper](https://wiki.archlinux.org/title/AUR_helpers), it's even simplier:
```
Comment (nnyyxxxx) [inline docs/index.md:44]:
```suggestion
If you use [yay](https://github.com/Jguer/yay), [paru](https://github.com/Morganamilo/paru) or any other [AUR Helper](https://wiki.archlinux.org/title/AUR_helpers), it's even simpler:
```

File: docs/index.md [hunk 2]
```diff
@@ -13,6 +15,55 @@ To get started, run the following command in your terminal:
...
+
+```bash
+paru -S linutil
```
Comment (nnyyxxxx) [inline docs/index.md:47]:
```suggestion
paru -S linutil
yay -s linutil
```

File: docs/index.md [hunk 3]
```diff
@@ -13,6 +15,55 @@ To get started, run the following command in your terminal:
...
+```
+
+Replace `paru` with your preferred helper and `linutil` with your preferred package.
```
Comment (nnyyxxxx) [inline docs/index.md:50]:
```suggestion
```

**Review by nnyyxxxx — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -24,6 +24,54 @@ curl -fsSL https://christitus.com/linux | sh
 ```bash
 curl -fsSL https://christitus.com/linuxdev | sh
 ```
+## ⬇️ Installation
+
+LinUtil is also available as a package in various repositories:
```
Comment (nnyyxxxx) [inline README.md:29]:
```suggestion
Linutil is also available as a package in various repositories:
```

File: README.md [hunk 5]
```diff
@@ -24,6 +24,54 @@ curl -fsSL https://christitus.com/linux | sh
...
+cd linutil
+makepkg -si
+```
+
+Replace `<package>` with your preferred package.
+
+If you use [yay](https://github.com/Jguer/yay), [paru](https://github.com/Morganamilo/paru) or any other [AUR Helper](https://wiki.archlinux.org/title/AUR_helpers), it's even simplier:
+
+```bash
+paru -S linutil
```
Comment (nnyyxxxx) [inline README.md:55]:
```suggestion
paru -S linutil
yay -S linutil
```

File: README.md [hunk 6]
```diff
@@ -24,6 +24,54 @@ curl -fsSL https://christitus.com/linux | sh
...
+```
+
+Replace `paru` with your preferred helper and `linutil` with your preferred package.
```
Comment (nnyyxxxx) [inline README.md:58]:
```suggestion
```

**Review by nnyyxxxx — CHANGES_REQUESTED** [review]

**Review by nnyyxxxx — APPROVED** [review]
If we're going to put html in the readme, might as well open a separate PR to add more of it for the header at the top.

**Review by ChrisTitusTech — APPROVED** [review]

**Pull request merged**